src/jobservice/main.go

```diff
@@ -60,9 +60,11 @@ func main() {
 		if utils.IsEmptyStr(secret) {
 			return nil, errors.New("empty auth secret")
 		}
-
-		adminClient := client.NewClient(config.GetAdminServerEndpoint(), &client.Config{Secret: secret})
-		jobCtx := impl.NewContext(ctx.SystemContext, adminClient)
+		remoteClient, err := remote.NewRemoteConfigDriver(config.GetAdminServerEndpoint(), &remote.Config{Secret: secret})
```

Comment:
shouldn't be the core endpoint? should we just drop the definition of adminserver at all?